contributors/guide/contributing.md

```diff
@@ -9,128 +9,224 @@ description: |
+correctly store data to or retrieve data from etcd. Integration tests are stored 
+in [kubernetes/test/integration/]. Running these can require the developer set up 
+additional functionality on their development system.
+* End-to-end ("e2e"): These are broad tests of overall system behavior and 
+coherence. These are more complicated as they require a functional kubernetes 
+cluster built from the sources to be tested. A 
+separate [document detailing e2e testing] and test cases themselves can be found 
+in [kubernetes/test/e2e/].
+* Conformance: These are a set of testcases, currently a subset of the 
+integration/e2e tests, that the Architecture SIG has approved to define the core 
+set of interoperable features that all Kubernetes deployments must support. For 
+more information on Conformance tests please see the [Conformance Testing] 
+Document.
+
+Continuous integration will run these tests either as pre-submits on PRs, 
+post-submits against /release branches, or both.  
+The results appear on [testgrid].
+
+SIG Testing is responsible for that official infrastructure and CI.
+The associated automation is tracked in the [test-infra repo].
+If you're looking to run e2e tests on your own infrastructure, [kubetest]
+is the mechanism.
+
+#### Security
+
+* [Security Release Page] - Outlines the procedures for the handling of security 
+  issues.
+* [Security and Disclosure Information] - Check this page if you wish to report 
+  a security vulnerability.
+
+#### Documentation
+
+* [Contributing to Documentation]
+
+#### Issues Management or Triage
+
+Have you ever noticed the total number of [open issues]?
+Helping to manage or triage these open issues can be a great contribution and a 
+great opportunity to learn about the various areas of the project. Triaging is 
+the word we use to describe the process of adding multiple types of descriptive 
+labels to GitHub issues, in order to speed up routing issues to the right folks.
+Refer to the [Issue Triage Guidelines] for 
+more information.
+
+[Contributor Experience SIG]: https://github.com/kubernetes/community/tree/master/sig-contributor-experience
+[community guiding principles]: /contributors/guide/expectations.md#code-review
+[contributor cheatsheet]: ./contributor-cheatsheet/
+[SIG]: #learn-about-sigs
+[standard lines of communication]: /communication/README.md
+[troubleshooting guide]: https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/
+[the GitHub workflow guide]: ./github-workflow.md
+[Kubernetes-specific GitHub Workflow]: ./pull-requests.md#the-testing-and-merge-workflow
+[github pull request]: https://help.github.com/articles/about-pull-requests/
+[the bot]: https://github.com/k8s-ci-robot
+[command reference documentation]: https://go.k8s.io/bot-commands
+[CLA page]: /CLA.md
+[Code Review]: #code-review
+[Learn About SIGs]: #learn-about-sigs
+[test flakes]: http://velodrome.k8s.io/dashboard/db/bigquery-metrics?orgId=1
```

Comment:
```suggestion
[test flakes]: /contributors/devel/sig-testing/flaky-tests.md#flaky-tests
```
References to velodrome were removed in https://github.com/kubernetes/community/pull/5193 It'd probably be better to link directly to the flakey test docs.